app/main.ipynb

```diff
@@ -56,33 +57,57 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
-   "outputs": [],
    "source": [
-    "# write your code here (for Python default syntax)"
+    "arr = [0] * 1000\n",
+    "sys.getsizeof(arr)"
```

Comment:
You have created a Python list with zeros and calculated its size using `sys.getsizeof()`. This is correct for the task requirement to use Python's default syntax.